.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
   - bower install
 
 script:
-  - ember l10n:convert -f false
+  - node scripts/l10n.js generate
```

Comment:
@niranjan94 While executing the script it throws error. Also even after generating the JSON the app still gives 404 error in the get requests.
![screenshot from 2017-06-21 03-38-45](https://user-images.githubusercontent.com/12807846/27358284-88206a8e-5633-11e7-8c6b-c1a1c2e1157e.png)